swig/python/CMakeLists.txt

```diff
@@ -117,6 +122,7 @@ endif ()
 
 function (symlink_or_copy source destination)
 
+  message(STATUS "Copying content of ${source} to ${destination}")
```

Comment:
to remove

Reply:
In MinGW command line environment,  the above code should be maintained.